.github/instructions/SqlServerDsc-guidelines.instructions.md

```
@@ -3,7 +3,16 @@
 applyTo: "**"
 ---
 
-# SqlServerDsc Guidelines
+# SqlServerDsc Requirements
+
+## Build & Test Workflow Requirements
+- Run in `pwsh`
+- Run scripts from project root
+- Setup build and test environment (once per `pwsh` session): `.build.ps1 -Task noop`
+- Build project: `.build.ps1 -Task build`
+- Run tests without coverage (wildcards allowed): `Invoke-PesterJob -Path '{tests filepath}' -SkipCodeCoverage`
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 111

(MD013, line-length)